src/Tribe/JSON_LD/Order.php

```diff
@@ -0,0 +1,149 @@
+<?php
+
+// Don't load directly
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+/**
+ * A JSON-LD class to hook and change how Orders work on Events
+ * @todo rework this class to make it standalone from The Events Calendar
+ */
+class Tribe__Tickets__JSON_LD__Order {
+
+	/**
+	 * Get (and instantiate, if necessary) the instance of the class
+	 *
+	 * @static
+	 * @return self
+	 *
+	 */
+	public static function instance() {
+		static $instance;
+
+		if ( ! $instance instanceof self ) {
+			$instance = new self;
+		}
+
+		return $instance;
+	}
+
+	/**
+	 * Value indicating low stock availability for a specific ticket.
+	 *
+	 * This can be overridden with the tribe_tickets_json_ld_low_inventory_level filter.
+	 *
+	 * @var int
+	 */
+	protected $low_stock = 5;
+
+	/**
+	 * Setup Google Event Data for tickets.
+	 */
+	public static function hook() {
```

Comment:
By having `hook()` been protected and on `instance()` I need to create a `lock` mechanism to prevent hooks from happening twice, Im going with a middle ground. Having `hook()` public but static and the constructor be protected.